src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -4,26 +4,36 @@
   },
   "dependencies": {
     "NuProj": "0.10.4-beta-gf7fc34e7d8",
-    "Microsoft.Build.Framework": "0.1.0-preview-00005",
-    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
-    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
+    "Microsoft.Build.Framework": "0.1.0-preview-00015",
```

Comment:
Please file an issue to remove tool-runtime\project.json and instead have the dependencies directly in BuildTools package.